src/views/Opportunities/OpportunityComponents/OpportunityForm.jsx

```diff
@@ -119,11 +119,10 @@ const OpportunityForm = (props) => {
           value: item.value,
         };
       }));
-
       setCompensationTypeOptions(data.paid.map((item) => {
         return {
           key: item.value,
-          label: item.value,
+          label: item.value == "Yes" ? "Paid":"unPaid" ,
```

Comment:
Minor formatting fix
```suggestion
          label: item.value == "Yes" ? "Paid" : "unPaid",
```